server/interfaces/HmppsUser.ts

```diff
@@ -0,0 +1,30 @@
+import CaseLoad from '../data/interfaces/prisonApi/CaseLoad'
+
+export type AuthSource = 'nomis' | 'delius' | 'external'
+
+export interface BaseUser {
+  authSource: AuthSource
+  username: string
+  userId: string
+  name: string
+  displayName: string
+  userRoles: string[]
+  token?: string
+}
+
+export interface PrisonUser extends BaseUser {
+  authSource: 'nomis'
+  staffId: number
+  activeCaseLoadId: string
+  caseLoads: CaseLoad[]
+}
```

Comment:
Hopefully this will force the consideration - is this a Prison user? Do they actually have the concept of a case load or a staffId etc.